examples/stable_diffusion_v2/ldm/models/diffusion/ddim.py

```diff
@@ -260,7 +260,7 @@ def p_sample_ddim(
                     c_in.append(ops.concat([unconditional_conditioning[i], c[i]], axis=0))
             else:
                 c_in = ops.concat([unconditional_conditioning, c], axis=0)
-            model_uncond, model_t = ops.split((self.model.apply_model(x_in, t_in, c_crossattn=c_in)), 0, 2)
+            model_uncond, model_t = ops.Split(0, 2)(self.model.apply_model(x_in, t_in, c_crossattn=c_in))
```

Comment:
Do we need to define this operator first (maybe in __init__ func) ?

Reply:
Not sure if this is a standard requirement, but I can do that.